modules/widgets/facebook-likebox.php

```diff
@@ -168,15 +168,19 @@ function form( $instance ) {
 		<p>
 			<label for="<?php echo esc_attr( $this->get_field_id( 'width' ) ); ?>">
 				<?php _e( 'Width in pixels', 'jetpack' ); ?>
-				<input type="number" class="smalltext" min="<?php echo esc_attr( $this->min_width ); ?>" max="<?php echo esc_attr( $this->max_width ); ?>" maxlength="3" name="<?php echo esc_attr( $this->get_field_name( 'width' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'width' ) ); ?>" value="<?php echo esc_attr( $like_args['width'] ); ?>" style="text-align: center;" />
+				<input type="number" class="smalltext" min="0" max="<?php echo esc_attr( $this->max_width ); ?>" maxlength="3" name="<?php echo esc_attr( $this->get_field_name( 'width' ) ); ?>" id="<?php echo esc_attr( $this->get_field_id( 'width' ) ); ?>" value="<?php echo esc_attr( $like_args['width'] ); ?>" style="text-align: center;" />
+				<br />
 				<small><?php echo sprintf( __( 'Minimum: %s', 'jetpack' ), $this->min_width ); ?> / <?php echo sprintf( __( 'Maximum: %s', 'jetpack' ), $this->max_width ); ?></small>
+				<br />
+				<small><?php _e( 'Enter 0 to adapt to container width.', 'jetpack' ); ?></small>
```

Comment:
Maybe we should escape this?

```suggestion
				<small><?php esc_html_e( 'Enter 0 to adapt to container width.', 'jetpack' ); ?></small>
```